airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -118,6 +134,69 @@ def data_field(self) -> str:
         """The name of the field in the response which contains the data"""
 
 
+class ShopifyDeletedEventsStream(ShopifyStream):
+
+    data_field = "events"
+    primary_key = "id"
+    deleted_event_key = "subject_id"
```

Comment:
how come this is an instance variable and not inlined into `produce_deleted_records_from_events`?